src/dist/component/package.rs

```diff
@@ -356,7 +358,16 @@ fn unpack_without_first_dir<'a, R: Read>(
 
         let size = entry.header().size()?;
         if size > MAX_FILE_SIZE {
-            return Err(format!("File too big {} {}", relpath.display(), size).into());
+            // If we cannot tell the user we will either succeed (great), or fail (and we may get a bug report), either
+            // way, we will most likely get reports from users about this, so the possible set of custom builds etc that
+            // don't report are not a great concern.
+            if let Some(notify_handler) = notify_handler {
+                notify_handler(Notification::Error(format!(
```

Comment:
Should this be a warning rather than an error?  In theory we may succeed anyway if swap helps out, and if so, an error seems risky.  Perhaps "Warning file {} is too large ({}) and may fail to unpack" ?

Reply:
I want people to tell us, so we can bump the number quickly :). Also its a smaller patch.

Reply:
Reasoning accepted.